backbone/src/main/java/org/researchstack/backbone/step/InstructionStep.java

```diff
@@ -1,16 +1,76 @@
 package org.researchstack.backbone.step;
 
+import android.util.Log;
+
+import org.researchstack.backbone.model.survey.InstructionSurveyItem;
+import org.researchstack.backbone.model.survey.SurveyItem;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.task.NavigableOrderedTask;
 import org.researchstack.backbone.ui.step.layout.InstructionStepLayout;
 
+import java.util.List;
+
 /**
  * An InstructionStep object gives the participant instructions for a task.
  * <p>
  * You can use instruction steps to present various types of content during a task, such as
  * introductory content, instructions in the middle of a task, or a final message at the completion
  * of a task.
  */
-public class InstructionStep extends Step
+public class InstructionStep extends Step implements NavigableOrderedTask.NavigationRule
```

Comment:
General FYI question: what would be the process to subclass InstructionStep w/ extra fields and possibly a different UI and then ensure that is used instead of InstructionStep?